### ISSUE TITLE
Make the terminal title bar show an untitled workspace when opening OR open with the most recently closed file

### ISSUE THREAD

**Issue by SharpBit** [CupOfGeo/adaptable-antelopes#13]
ON APPLICATION STARTUP

When opening the application with a new file, show `ThoughtBox- Untitled` in the terminal title bar
- Note: may not be needed if there is no way to disable the welcome screen (see #70)

When opening an existing file, show `ThoughtBox - {file_path}` in the title bar

**Comment by SharpBit** [comment]
Similar to issue #6

**Comment by BoraxTheClean** [comment]
This seems to be blocked by (or a duplicate of) https://github.com/CupOfGeo/adaptable-antelopes/issues/7

Since we need to determine how to save this state and load it when the application starts.

**Comment by ntdkhiem** [comment]
Solved displaying file's name in the title bar. https://github.com/BoraxTheClean/adaptable-antelopes/commit/bbac56c53743f9b76724af82d2a50f40ea4981e0

**Comment by SharpBit** [comment]
Still need to make it show as soon as the application opens, right now it only shows if you save the file or open one first (or make a new file.)
